include/chemfiles/capi/topology.h

```diff
@@ -230,6 +230,43 @@ CHFL_EXPORT chfl_status chfl_topology_residues_linked(
     bool* result
 );
 
+/// Add a bond between the atoms at indexes `i` and `j`
+/// with bond order `bond_order` in the `topology`.
+///
+/// @example{tests/capi/doc/chfl_topology/bond_order.c}
+/// @return The operation status code. You can use `chfl_last_error` to learn
+///         about the error if the status code is not `CHFL_SUCCESS`.
+CHFL_EXPORT chfl_status chfl_topology_add_bond_w_order(
```

Comment:
I am a bit bothered by the single `w` here, but `chfl_topology_add_bond_with_order` would be over the 31 characters limit.

What do you think of `chfl_topology_bond_with_order`?

Reply:
And same for `chfl_frame_bond_with_order` :smiley: